odoosync.php

```diff
@@ -185,4 +177,20 @@ function odoosync_civicrm_post($op, $objectName, $objectId, &$objectRef) {
     $lineItem->process();
   }
 
+  //Contact
+  if ($objectName = 'Contact'
+    && ($op == 'create' || $op == 'edit' || $op == 'delete'|| $op == 'trash'|| $op == 'restore')
+  ) {
+    if (isset($objectRef->contact_id)) {
```

Comment:
instead of writing this logic here, maybe u should keep conisisnted which what u wrote for the Contribution, EntityFinancialTrxn and LineItem entities above and move it to CRM_Odoosync_Hook_Post_Contact class